feature/experimental/p4rt/otg_tests/traceroute_packetin_test/traceroute_packetin_test.go

```diff
@@ -343,24 +363,28 @@ func (traceroute *TraceroutePacketIO) GetPacketTemplate() *PacketIOPacket {
 	return &traceroute.PacketIOPacket
 }
 
-func (traceroute *TraceroutePacketIO) GetTrafficFlow(ate *ondatra.ATEDevice, isIpv4 bool, TTL uint8, frameSize uint32, frameRate uint64) []gosnappi.Flow {
+func (traceroute *TraceroutePacketIO) GetTrafficFlow(ate *ondatra.ATEDevice, dstMac string, isIpv4 bool, TTL uint8, frameSize uint32, frameRate uint64) []gosnappi.Flow {
 	if isIpv4 {
 		flow := gosnappi.NewFlow()
```

Comment:
Need frame size/rate set for both of these flows.

Reply:
Sorry, missed to add them. Added framesize/rate